client/platform/desktop/frontend/components/MultiTrainingMenu.vue

```diff
@@ -0,0 +1,253 @@
+<script lang="ts">
+import type { DataTableHeader, DataOptions } from 'vuetify';
+
+import {
+  computed, defineComponent, onBeforeMount, set, del, reactive, watch, toRef,
+} from '@vue/composition-api';
+import { DatasetMeta, TrainingConfigs, useApi } from 'viame-web-common/apispec';
+import { listDatasets } from '../store/dataset';
+
+export default defineComponent({
+  setup(_, { root }) {
+    const { getTrainingConfigurations, runTraining } = useApi();
+
+    const data = reactive({
+      items: [] as DatasetMeta[],
+      stagedItems: {} as Record<string, DatasetMeta>,
+      trainingOutputName: '',
+      selectedTrainingConfig: 'foo.whatever',
+      trainingConfigurations: {
+        configs: ['foo.whatever', 'bar.somethingelse'],
```

Comment:
might want to change these.  If no VIAME Toolkit is found or fails to initialize these are the configs that are returned.  There is an additional comment about handling the error returned from `getTrainingConfigurations`.  When that happens we probably want training disabled and not these items displayed.

Reply:
I don't where that additional comment went,  But it was about the fact that if `kwiver` isn't found or there is no VIAME Toolkit we should indicate clearly on the Training page that they need to install VIAME Toolkit to utilize training.